v19.2/vectorized-execution.md

```diff
@@ -0,0 +1,58 @@
+
+
+## Configuring vectorized execution
+
+By default, vectorized execution is enabled in CockroachDB for all supported "streaming" SQL operations (i.e. operations that do not require [memory buffering](https://en.wikipedia.org/wiki/Data_buffer)). Non-streaming operations include global [sorts](query-order.html), [window functions](window-functions.html), [hash joins](joins.html#hash-joins), and [merge joins](joins.html#merge-joins) on non-unique columns. Merge joins on columns that are guaranteed to have one row per value, also known as "key columns", are considered streaming operations.
+
+You can turn vectorized execution on or off for all operations with the `sql.defaults.vectorize` [cluster setting](cluster-settings.html) or the `vectorize` [session variable](set-vars.html). The following options are supported for both the cluster setting and session variable:
+
+Option | Description
+----------|------------
+`auto` | Instructs CockroachDB to use the column-oriented vectorized SQL execution engine on streaming operations. By default, `vectorize=auto`.
+`experimental_on` | Turns vectorization on for all supported operations. We do not recommend using this setting in production environments, as it can lead to memory issues. See [Known Limitations](vectorized-execution.html#known-limitations) for more information.
+`off` | Turns vectorization off for all operations.
+
+You can verify whether will use the execution engine or not with the [`EXPLAIN`](explain.html) statement.
+
+For more information about configuring cluster settings, see [Cluster Settings](cluster-settings.html). For more information about setting session variables, see [`SET` &lt;session variable&gt;](set-vars.html).
+
+### Setting the row threshold for vectorized execution
+
+The efficiency of column-oriented execution increases with the number of rows processed. If you are querying a table with a small number of rows, it is currently more efficient to use row-oriented execution.
+
+With `vectorize=auto`, the `vectorize_row_count_threshold` [cluster setting](cluster-settings.html) specifies the minimum number of rows required to use the vectorized engine to execute a query plan. If the number of rows being processed falls below the row count threshold, CockroachDB uses the row-oriented execution engine instead. By default, `vectorize_row_count_threshold=1000`.
```

Comment:
Also, is the `vectorize_row_count_threshold` cluster setting still respected when `vectorize=experimental_on`?  or does it ignore `v_r_c_t` altogether when `vectorize=experimental_on` since presumably the latter means "fuck it we're going five blades", right?  It's implied here (I think?) but we should make it explicit.